pkg/indicator/sma.go

```diff
@@ -30,6 +30,33 @@ func (inc *SMA) Last() float64 {
 	return inc.Values[len(inc.Values)-1]
 }
 
+func (inc *SMA) Max() float64 {
+	if len(inc.Values) == 0 {
+		return 0.0
+	}
+	max := inc.Values[len(inc.Values)-1]
```

Comment:
Why not pick the first as the default number

Reply:
both are ok
which one is better?

Reply:
It will be faster if you could pick up the first